easybuild/easyconfigs/g/gff3tool/gff3tool-2.1.0-foss-2022a.eb

```
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'gff3tool'
+version = '2.1.0'
+
+homepage = 'https://gff3toolkit.readthedocs.io'
+description = 'Python programs for processing GFF3 files'
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+sources = ['gff3tool-2.1.0.tar.gz']
```

don't hardcode the version here, use `%(version)s` template:

```suggestion
sources = ['gff3tool-%(version)s.tar.gz']